src/components/SecretsModal/UniversalFields.js

```diff
@@ -54,11 +65,11 @@ const UniversalFields = props => {
           component: 'accessTo'
         }}
         items={[
-          { id: 'git', text: 'Git Server', component: 'accessTo' },
-          { id: 'docker', text: 'Docker Registry', component: 'accessTo' }
+          { id: 'git', text: 'Git Server' },
+          { id: 'docker', text: 'Docker Registry' }
```

Comment:
We should document that these secrets are only for resources that require the tekton-labelled secrets, and that plain kube secrets required by other resource types (e.g. cluster, pullRequest, storage) are not covered by this UI.